scripts/infra/README.md

```diff
@@ -49,6 +50,8 @@ Either way, you'll need to edit the config to reference your cloud environment(s
 Example:
 
 ```bash
+# Ensure proper system packages are installed on Fedora/RHEL/CentOS
```

Comment:
Probably need more of a warning that this is only applicable if your local env is Fedora/RHEL/CentOS

Reply:
Added some addn text here, lmk what you think